src/Configurator/CopyFromPackageConfigurator.php

```diff
@@ -12,41 +12,45 @@
 namespace Symfony\Flex\Configurator;
 
 use LogicException;
+use Symfony\Flex\Lock;
 use Symfony\Flex\Recipe;
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
  */
 class CopyFromPackageConfigurator extends AbstractConfigurator
 {
-    public function configure(Recipe $recipe, $config, array $options = [])
+    public function configure(Recipe $recipe, $config, Lock $lock, array $options = [])
     {
         $this->write('Setting configuration and copying files');
         $packageDir = $this->composer->getInstallationManager()->getInstallPath($recipe->getPackage());
-        $this->copyFiles($config, $packageDir, $this->options->get('root-dir'), $options['force'] ?? false);
+        $options = array_merge($this->options->toArray(), $options);
+
+        $this->copyFiles($config, $packageDir, $options);
     }
 
-    public function unconfigure(Recipe $recipe, $config)
+    public function unconfigure(Recipe $recipe, $config, Lock $lock)
     {
         $this->write('Removing configuration and files');
         $packageDir = $this->composer->getInstallationManager()->getInstallPath($recipe->getPackage());
         $this->removeFiles($config, $packageDir, $this->options->get('root-dir'));
     }
 
-    private function copyFiles(array $manifest, string $from, string $to, bool $overwrite = false)
+    private function copyFiles(array $manifest, string $from, array $options)
     {
+        $to = $options['root-dir'] ?? getcwd();
```

Comment:
not sure about this: getcwd introduces external state - can't we remove it from the PR altogether?

Reply:
Hum, we use `'root-dir' => $extra['symfony']['root-dir'] ?? '.'` elsewhere so my argument is void :)
Shouldn't we use the same?